package.json

```diff
@@ -48,7 +48,8 @@
     "jest": "26.6.3"
   },
   "resolutions": {
-    "@changesets/git": "1.1.1"
+    "@changesets/git": "1.1.1",
+    "graphql": "15.4.0-experimental-stream-defer.1"
```

Comment:
This is a bit annoying but required in order to prevent the graphql.js from mixed realms issue.

Unfortunately, the stream-defer release line is not kept up2sync with main, there is currently no 15.5.x release available.

Reply:
Got it, but it means that the entire repo is using this experimental version, so maybe we can detach the example directory from the monorepo workspace?

Reply:
I rebase on main and detached the helix example from the main repo, another drawback is that we cannot simply link envelope core as it results in the graphql from two realm issue again, which is a bit of a bummer when you want to make changes in envelop and see how it affects the example. But I guess then you can temporarily add the resolutions entry for the desired graphql version and then use yarn link.

Reply:
Yeah, that's fine :) I think we can merge this one as-is, just to have it as a reference and we can later see if this could be improved (maybe with pnpm?)